apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -460,13 +469,15 @@ export class Utilities {
   private static _executeCommandInternal(
     command: string, args: string[], workingDirectory: string,
     stdio: (string|number)[] | undefined,
-    environment?: IEnvironment): child_process.SpawnSyncReturns<Buffer> {
+    environment?: IEnvironment,
+    keepEnvironment: boolean = false
```

Comment:
Since this is a private helper, neither of these parameters should be optional.  It might be safer for the caller to always pass them to make sure it didn't forget anything.